src/components/dialogs/parameters/non-evacuated-energy/columns-definitions.ts

```diff
@@ -0,0 +1,266 @@
+    {
+        label: 'StageName',
+        dataKey: NAME,
+        initialValue: '',
+        editable: false,
+        textItems: true,
+        width: '60%',
+    },
+    {
+        label: 'Active',
+        dataKey: ACTIVATED,
+        initialValue: true,
+        checkboxItems: true,
+        editable: true,
+        width: '10%',
+    },
+];
+
+export const COLUMNS_DEFINITIONS_GENERATORS_CAPPINGS = [
+    {
+        label: 'GeneratorsCappingsKind',
+        dataKey: GENERATORS_CAPPINGS_KIND,
+        equipmentTypes: GENERATION_KINDS,
+        initialValue: GENERATION_KINDS[0].id,
+        editable: true,
+        menuItems: true,
+        width: '20%',
+    },
+    {
+        label: 'GeneratorsCappingsFilter',
+        dataKey: GENERATORS_CAPPINGS_FILTER,
+        initialValue: [],
+        editable: true,
+        directoryItems: true,
+        equipmentTypes: INJECTIONS_EQUIPMENT_TYPES,
+        elementType: elementType.FILTER,
+        titleId: 'FiltersListsSelection',
+        width: '40%',
+    },
+    {
+        label: 'Active',
+        dataKey: ACTIVATED,
+        initialValue: true,
+        checkboxItems: true,
+        editable: true,
+        width: '10%',
+    },
+];
+
+export const COLUMNS_DEFINITIONS_MONITORED_BRANCHES = [
+    {
+        label: 'MonitoredBranchesFilter',
+        dataKey: BRANCHES,
+        initialValue: [],
+        editable: true,
+        directoryItems: true,
+        equipmentTypes: MONITORED_BRANCHES_EQUIPMENT_TYPES,
+        elementType: elementType.FILTER,
+        titleId: 'FiltersListsSelection',
+        width: '30%',
+    },
+    {
+        label: 'istN',
+        dataKey: MONITORED_BRANCHES_IST_N,
+        initialValue: true,
+        checkboxItems: true,
+        editable: true,
+        width: '10%',
+    },
+    {
+        label: 'limitNameN',
+        dataKey: MONITORED_BRANCHES_LIMIT_NAME_N,
+        initialValue: '',
+        editable: true,
+        textItems: true,
+        width: '10%',
+    },
+    {
+        label: 'coeffN',
+        dataKey: MONITORED_BRANCHES_COEFF_N,
+        initialValue: 100,
+        editable: true,
+        floatItems: true,
+        width: '10%',
+    },
+    {
+        label: 'istNm1',
+        dataKey: MONITORED_BRANCHES_IST_N_1,
+        initialValue: true,
+        checkboxItems: true,
+        editable: true,
+        width: '10%',
+    },
+    {
+        label: 'limitNameNm1',
+        dataKey: MONITORED_BRANCHES_LIMIT_NAME_N_1,
+        initialValue: '',
+        editable: true,
+        textItems: true,
+        width: '10%',
+    },
+    {
+        label: 'coeffNm1',
+        dataKey: MONITORED_BRANCHES_COEFF_N_1,
+        initialValue: 100,
+        editable: true,
+        floatItems: true,
+        width: '10%',
+    },
+    {
+        label: 'Active',
+        dataKey: ACTIVATED,
+        initialValue: true,
+        checkboxItems: true,
+        editable: true,
+        width: '10%',
+    },
+];
```

Comment:
Isn't it that some of this fields should enable/disable other ones ? Like setting istN should disable limitNameN or coeffN ?